pkg/report/linux.go

```diff
@@ -76,6 +76,7 @@ func ctorLinux(cfg *config) (reporterImpl, []string, error) {
 		regexp.MustCompile(`^mm/util.c`),
 		regexp.MustCompile(`^kernel/rcu/.*`),
 		regexp.MustCompile(`^arch/.*/kernel/traps.c`),
+		regexp.MustCompile(`^arch/.*/kernel/unwind_orc.c`),
```

Comment:
Only x86 has ORC unwinder, so using * for arch but _orc.c is not useful.
And we also need to ignore all other x86 unwinders, not just ORC.

```
arch/arc/kernel/unwind.c
arch/arm/kernel/unwind.c
arch/ia64/kernel/unwind.c
arch/ia64/kernel/unwind_decoder.c
arch/ia64/kernel/unwind_i.h
arch/loongarch/kernel/unwind.c
arch/loongarch/kernel/unwind_guess.c
arch/loongarch/kernel/unwind_prologue.c
arch/microblaze/kernel/unwind.c
arch/openrisc/kernel/unwinder.c
arch/parisc/kernel/unwind.c
arch/s390/kernel/unwind_bc.c
arch/sh/kernel/unwinder.c
arch/x86/kernel/unwind_frame.c
arch/x86/kernel/unwind_guess.c
arch/x86/kernel/unwind_orc.c
```